docs/product-oracle-weblogic-installation-and-configuration-guide/src/main/asciidoc/chap-configure.adoc

```diff
@@ -20,35 +20,11 @@ USER_MEM_ARGS=-Xms512m -Xmx1024m -XX:MaxPermSize=512m
 
 The following custom properties require configuration on the WebLogic Server:
 
-*Properties Required for Business Central and {KIE_SERVER}*
-
-* `org.kie.executor.jms.cf`: JNDI name of the connection factory for sending messages with job executor requests.
-
-* `org.kie.executor.jms.queue`: JNDI name of the destination (queue) for sending messages with job executor requests.
-
-*Properties Required for Business Central*
-
-ifdef::BA[]
-* `kie.services.jms.queues.response`:
-JNDI name of the response queue for JMS remote API of Business Central. Set to `jms/KIE.RESPONSE.ALL`.
-endif::BA[]
-
-ifdef::DM[]
-* `org.jbpm.designer.perspective=ruleflow`:
-Changes the default perspective in the designer to `RuleFlow` instead of `Full`.
-endif::DM[]
-
-* `org.uberfire.start.method`:
-Defines startable beans for Uberfire. Set to `ejb` (Enterprise Java Beans).
-
-* `org.uberfire.domain`:
-Sets the domain for Uberfire to use. Set to `OracleDefaultLoginConfiguration`.
+*Properties Required for {KIE_SERVER}*
 
 * `com.sun.jersey.server.impl.cdi.lookupExtensionInBeanManager`:
 Uses `BeanManager` to look up extensions. This avoids conflicts across multiple instances. Set to `true`.
 
-*Properties Required for {KIE_SERVER}*
-
 * `kie.server.jms.queues.response`:
 JNDI name of the response queue for the {KIE_SERVER}.  Set to `jms/KIE.SERVER.RESPONSE`.
```

Comment:
Please add here also these system properties 
org.jbpm.server.ext.disabled=true 
org.jbpm.ui.server.ext.disabled=true
org.jbpm.case.server.ext.disabled=true
They should disable jBPM functionality of Decision Server, so users won't see any error message.
Add them also to the example below.